### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by rajgodwin**
Added 2.2.0-preview version release listing the new features rolled out

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
 
 | Date | Version | Updates |
 |------- | -------------------- | ------------------------------ |
+| April 2021 | [2.2.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.2.0-preview) | <li>Fixes for integrated windows authentication to use -AccountId with Connect-MicrosoftTeams</li><li>Added cmdlet to get details of total change Notification event that can be sent and user</li><li>Added cmdlet to get multi-geo region for users and groups</li><li>Handling of values passed to TeamsEnvironment name was case sensitive. This has been fixed.</li><li>Major refactor of remote session management within the module to facilitate unit tests. There should be no functional change for tenant admins.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| April 2021 | [2.2.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.2.0-preview) | <li>Fixes for integrated Windows authentication to use -AccountId with Connect-MicrosoftTeams</li><li>Added cmdlet to get details of total change notification events that can be sent to users</li><li>Added cmdlet to get multi-geo region for users and groups</li><li>Handling of values passed to TeamsEnvironment name was case sensitive. This has been fixed.</li><li>Major refactor of remote session management within the module to facilitate unit tests. There should be no functional change for tenant admins.</li>|
```

Comment (JohanFreelancer9) [reply Teams/teams-powershell-release-notes.md:29]: Not sure of these suggestions - please confirm.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**